docs/history.md

```diff
@@ -4,7 +4,8 @@
 
 #### Breaking Changes
 
-N/A
+* `accounts-2fa@1.0.1`
```

Comment:
Since there is a breaking change, the package's major version should be increased.